02_bubble/chitwanAgrawal/02_bubble.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	for i := 1; i < len(s); i++ {
```

Comment:
Create a copy of `s` and modify the copy.
Create a test ensuring that `s` is not modified after execution of `bubble(s)`